tests/integration/annotation_import/test_send_to_annotate_mea.py

```diff
@@ -6,13 +6,15 @@
 
 def test_send_to_annotate_from_model(client, configured_project,
                                      model_run_predictions,
-                                     model_run_with_data_rows,
-                                     project_with_ontology):
+                                     model_run_with_data_rows, project):
     model_run = model_run_with_data_rows
     data_row_ids = [p['dataRow']['id'] for p in model_run_predictions]
     assert len(data_row_ids) > 0
 
-    destination_project, _ = project_with_ontology
+    destination_project = project
```

Comment:
does it make any sense to setup a fixture to wrap these 4 lines? I think it is the same code as below